docs/dg/architecture.md

```diff
@@ -65,9 +65,12 @@
 ## ReportGenerator(Main)
 
 [`ReportGenerator`](https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/report/ReportGenerator.java)
- 1. uses `GitClone` API to clone the repository from *GitHub*.
- 1. copies the template files into the designated output directory.
- 1. uses `CommitReporter` and `AuthorshipReporter` to produce the commit and authorship summary, respectively.
+ 1. clones repositories from *GitHub* using the `GitClone` API, in a multi-threaded fashion.
+    * by default, 4 threads are used for cloning; the number of threads can be specified using the CLI argument `--cloning-threads <threads>`.
```

Comment:
The sub-bullets should capitalise the first letter. "By default" instead of "by default".